xmtp_mls/src/builder.rs

```diff
@@ -238,7 +260,7 @@ mod tests {
                 .unwrap();
 
         let client_a = ClientBuilder::new(wallet.clone().into())
-            .local_grpc()
+            .local_grpc(5556)
```

Comment:
Nothing blocking, but I have a small nit request to put static-ish values like `port`s and `url`s and anything resembling configuration-like values be moved to the top of the file and be made into `const FOO = ...`. and referenced by name rather than hard-coding values, anytime that value is used/declared in > 1 place.

Reply:
I didn't actually mean to commit this set of file changes at all. Will revert